src/core/Track.cpp

```diff
@@ -3042,3 +3043,30 @@ void TrackView::createTCOView( TrackContentObject * tco )
 	}
 	tco->selectViewOnCreate( false );
 }
+
+
+/*! \brief Show a message box warning the user that this track is about to be closed */
+bool TrackOperationsWidget::warnRemoveTrack()
+{
+    QString messageRemoveTrack = tr("After removing a track, it can not"
+                                    "be recovered. Are you sure you want to remove track \"");
+    messageRemoveTrack += m_trackView->getTrack()->name() + "\"?";
+    QString messageTitleRemoveTrack = tr("Removing this track");
```

Comment:
I think `Confirm removing track` will be slightly better.

Reply:
I would go with `Confirm removal`. Or, if you really feel like "track" needs to be there, `Confirm track removal`. The latter seems redundant to me though, the user knows they clicked on a track-related action and the message content specifies which track will be removed.

Reply:
Anyway, this title really needs a change.

Reply:
@PhysSong latest commit has it changed already

Reply:
I can only see the change in the function name, not the message box title.

Reply:
> I can only see the change in the function name, not the message box title.

oh you were referring to the message box title. Should it be "Confirm removal" as well?